lib/github_api/params_hash.rb

```diff
@@ -9,7 +9,7 @@ class ParamsHash < DelegateClass(Hash)
     include Normalizer
     include MimeType
 
-    REQUEST_PARAMS = [:accept, :media, :data, :raw, :content_type, :headers]
+    REQUEST_PARAMS = [:accept, :media, :data, :raw, :content_type, :headers, :options]
```

Comment:
I want to send a header with the HTTP request.

Not append a `header` param to the GET url.